android_pre_post_test.py

```diff
@@ -8,18 +8,19 @@
     get_battery_info,
     parse_battery_info,
     wait_for_drop,
+    start_test
 )
 from utils import finish_same_line, write_same_line
 
-OUTPUT = "/home/sparky/Documents/mozwork/"
```

Comment:
🎆 🍾 🎉 

Heh, j/k - easy to fix, and this never bothered me, but thanks for fixing so soon!